reactive/ovn_chassis_charm_handlers.py

```diff
@@ -181,3 +181,54 @@ def provide_chassis_certificates_to_principal():
                             level=ch_core.hookenv.WARNING)
 
     reactive.clear_flag('ovn.certs.changed')
+
+
+@reactive.when_any('config.changed.ovs-exporter-channel',
```

Comment:
The reactive handlers in this layer are opt-in, and all of them should be guarded with:
```python
reactive.when(OVN_CHASSIS_ENABLE_HANDLERS_FLAG, ...
```

You probably also want to assert that the charm is installed by looking for the `charm.installed` flag?

We also do not want to perform any work when in update-status hook, so all handlers should have:
```python
@reactive.when_none('is-update-status-hook', ...
```